community/mosh/depends

```diff
@@ -0,0 +1,3 @@
+openssh
+perl
```

Comment:
Does the package need perl or is it just a build time dependency ?

Reply:
yes, /usr/bin/mosh is written in perl, so it is needed for mosh to function